services/dynamic-scheduler/docker/healthcheck.py

```diff
@@ -6,9 +6,10 @@
     COPY --chown=scu:scu docker/healthcheck.py docker/healthcheck.py
     HEALTHCHECK --interval=30s \
                 --timeout=30s \
-                --start-period=1s \
+                --start-period=20s \
+                --start-interval=1s \
                 --retries=3 \
-                CMD python3 docker/healthcheck.py http://localhost:8000/
+                CMD python3 docker/healthcheck.py http://localhost:8080/v0/
```

Comment:
```suggestion
                CMD python3 docker/healthcheck.py http://localhost:8000/health
```

Reply:
why? thi is not the way it was defined... unless that changed last week